lua/SS13.lua

```diff
@@ -0,0 +1,79 @@
+local SS13 = {}
+
+SS13.SSlua = dm.global_vars:get_var("SSlua")
+
+SS13.global_proc = "some_magic_bullshit"
+
+local states = SS13.SSlua:get_var("states"):to_table()
+for _, state in pairs(states) do
```

Comment:
Since we're on Luau, you don't need `pairs` and can just do this:

```suggestion
for _, state in states do
```

It's marginally faster but also cleaner.

I am under the assumption we are on the latest version of Luau.

Reply:
Not sure if Mlua is on the latest version, but I can check.